src/SldStyleParser.ts

```diff
@@ -510,404 +518,463 @@ export class SldStyleParser implements StyleParser<string> {
   }
 
   /**
-   * Get the GeoStyler-Style Filter from an SLD Rule.
+   * Creates a geostyler-style FunctionFilter from a SLD Function.
    *
-   * Currently only supports one Filter per Rule.
-   *
-   * @param {object} sldRule The SLD Rule
-   * @return {Filter} The GeoStyler-Style Filter
+   * @param sldFilter The SLD Filter
+   * @return The geostyler-style FunctionFilter
    */
-  getFilterFromRule(sldRule: any): Filter | undefined {
-    const {
-      Filter: sldFilters
-    } = sldRule;
-    if (!sldFilters) {
-      return;
-    }
-    const comparison = _get(sldFilters, '[0].$$[0]');
-    if (!comparison) {
-      return;
-    }
-    const operator = comparison['#name'];
-    if (!operator) {
-      return;
+  getFunctionFilterFromSldFilter(sldFilter: any): GeoStylerFunction | undefined {
+    // TODO:
```

Comment:
Is this still relevant?

Reply:
yes. i'll add some text to make this more obvious.